packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  h,
+  Prop,
+  State,
+} from '@stencil/core';
+
+@Component({
+  tag: 'va-breadcrumbs',
+  styleUrl: 'va-breadcrumbs.css',
+  shadow: true,
+})
+export class VaBreadcrumbs {
+  @Element() el: HTMLElement;
+
+  /**
+   * Adds an aria-label attribute to the <nav /> element.
+   */
+  @Prop() label: string = 'Breadcrumb';
+
+  /**
+   * Analytics tracking function(s) will not be called
+   */
+  @Prop() disableAnalytics: boolean;
+
+  @State() breadcrumbs: Array<Node>;
+
+  @Event({
+    eventName: 'component-library-analytics',
+    composed: true,
+    bubbles: true,
+  })
+  componentLibraryAnalytics: EventEmitter;
+
+  private fireAnalyticsEvent(event: MouseEvent): void {
+    if (!this.disableAnalytics) {
+      const target = event.target as HTMLElement;
+      // If it's a link being clicked, dispatch an analytics event
+      if (target?.tagName === 'A') {
+        const details = {
+          componentName: 'va-breadcrumbs',
+          action: 'linkClick',
+          details: {
+            clickLabel: target.innerText.trim(),
+            clickLevel: parseInt(target.dataset.index, 10) + 1,
```

Comment:
I think we might be able to get away with not setting `data-index` and rely on some [array functionality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex), maybe like:

```js
clickLabel: this.breadcrumbs.findIndex(node => node.innerText === target.innerText) + 1,
```

(I haven't tested this)

That does add some looping, but it prevents stuff being added to the DOM which could become tech debt in the future and puzzle developers (like the `mobileFirstProp` thing).

Reply:
This suggestion could also be problematic if two breadcrumb pieces had the same text which of course _shouldn't_ happen, but you never know. @k80bowman might have some thoughts on what to do.

Reply:
I do like the idea of not setting data index. But I think we need to be clear on what kind of logic we are expecting here. It also might be worth breaking that logic out into its own function. Let me see if I understand this correctly...

For analytics, they need a clickLevel value that maps to the index of whatever breadcrumb is clicked on in the array of breadcrumbs (using 1-index, not 0-index). The way we have it now, we are attaching a data-index value to the breadcrumb link element that the analytics function can then retrieve and assign to clickLevel.

Brooks, your suggestion is to not add the data-index value to the breadcrumb link element and instead use `findIndex` to locate the index for the node that is clicked on by comparing the inner text from the target node to the array of breadcrumb text values.

If the concern is that using `innerText` could be problematic because of repeat breadcrumb names, could we use a different value that is less likely to repeat? I wonder if href would be better.

Reply:
And if we use href, I would definitely recommend breaking this logic out into its own function. Something like

```
const getClickLevel = (breadcrumbs, target) => {
  const targetHref = target.attributes['href'].value;
  const index = breadcrumbs.findIndex(node => node.href === targetHref;
  return index + 1;
}
```

This way we can make it very clear what we are doing and hopefully prevent any future confusion.

Reply:
same caveat as Brooks on this code, though. I have not tested it.

Reply:
Comparing `href` as a way of finding the index does sound better. We may also be able to compare the `target` and `node` (or whatever we name the param for the iterator) objects directly, but I'm not sure if that would do what we want.

Reply:
Comparing `target` and `node` sounds like a good thing to try first. But then if that doesn't do what you want, then maybe try comparing `href`s.

Reply:
I tried the following and I'm getting the correct value for click level, but one of my tests fails due to never receiving an event. If I change it to what is currently pushed up, the test passes.
```javascript
  private getClickLevel(target: HTMLAnchorElement) {
    const anchorNodes = Array.from(this.el.querySelectorAll('a'));
    const index = anchorNodes.findIndex((node: HTMLAnchorElement) =>
      node.isEqualNode(target),
    );
    return index + 1;
  }
```

Reply:
I managed to get it working, but I'm having some weird behavior when modifying the test to check for the last anchor element. There are 3 anchorElements, but the test fails because it never receives an event if I select anchorElements[2] and call click on it.

This works:
```javascript
    const anchorElements = await page.findAll('va-breadcrumbs >>> li > a');
    await anchorElements[1].click();
```

This doesn't work:
```javascript
    const anchorElements = await page.findAll('va-breadcrumbs >>> li > a');
    await anchorElements[2].click();
```